data/converters/converters.go

```diff
@@ -0,0 +1,195 @@
+var BoolNOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeBool,
+}
+
+// Float64NOOP The input and output values are both int64.
+var Float64NOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeFloat64,
+}
+
+// StringNOOP The input and output values are both string.
+var StringNOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeString,
+}
+
+// AnyToNullableString Converts any non-null value into a string.
+var AnyToNullableString = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableString,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		str := fmt.Sprintf("%v", v)
+		return &str, nil
+	},
+}
+
+// AnyToString Converts any value into a string.
+var AnyToString = data.FieldConverter{
+	OutputFieldType: data.FieldTypeString,
+	Converter: func(v interface{}) (interface{}, error) {
+		return fmt.Sprintf("%v", v), nil
+	},
+}
+
+// Float64ToNullableFloat64 returns an error if the input is not a float64.
+var Float64ToNullableFloat64 = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableFloat64,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		val, ok := v.(float64)
+		if !ok {
+			return nil, toConversionError("float64", v)
+		}
+		return &val, nil
+	},
+}
+
+// Int64ToNullableInt64 returns an error if the input is not an int64.
+var Int64ToNullableInt64 = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableInt64,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		val, ok := v.(int64)
+		if !ok {
+			return nil, toConversionError("int64", v)
+		}
+		return &val, nil
+	},
+}
+
+// Uint64ToNullableUInt64 returns an error if the input is not a uint64.
+var Uint64ToNullableUInt64 = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableUint64,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		val, ok := v.(uint64)
+		if !ok {
+			return nil, toConversionError("uint64", v)
+		}
+		return &val, nil
+	},
+}
+
+// BoolToNullableBool returns an error if the input is not a bool.
+var BoolToNullableBool = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableBool,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		val, ok := v.(bool)
+		if !ok {
+			return nil, toConversionError("bool", v)
+		}
+		return &val, nil
+	},
+}
+
+// TimeToNullableTime returns an error if the input is not a time.Time.
+var TimeToNullableTime = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableTime,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		val, ok := v.(time.Time)
+		if !ok {
+			return nil, toConversionError("time.Time", v)
+		}
+		return &val, nil
+	},
+}
+
+// RFC3339StringToNullableTime convert a string with RFC3339 to a time object.
```

Comment:
this would be *time.Time.

Also seems odd that sometimes we are creating *time.Time and other times non-pointer time.Time.

Reply:
i removed that one for now -- there will likely be another round as i try to actually replace the 5 copies of the file we have :)